packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -143,7 +146,10 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     public readonly routerPath: string,
     private readonly log: Logger,
     private readonly enhanceWithContext: ContextEnhancer<any, any, any, any, any>,
-    private readonly options: RouterOptions = { isDev: false, isServerless: false }
+    private readonly options: RouterOptions = {
+      isDev: false,
+      versionedRouteResolution: 'oldest',
+    }
```

Comment:
NIT: I know it was done to avoid having to change all the constructor calls, but I wonder how safe having this `options` be an optional parameter is. I don't think we're using the `Router` in that many places, so maybe making `options` mandatory now could make sense, WDYT?

Reply:
Sounds good to me!